src/PerformanceTests/Tests/Categories/TestSessionInitializer.cs

```diff
@@ -0,0 +1,15 @@
+namespace Categories
+{
+    using System;
+    using NUnit.Framework;
+
+    [SetUpFixture]
+    public class TestSessionInitializer
+    {
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            Base.SessionId = DateTime.UtcNow.Ticks.ToString();
```

Comment:
@SimonCropp LOL, I refactored it to that at first. Then changed it back as to keep current code similar.